### PR TITLE
FAQ Section Added - Which provides answers to the Frequently Asked Questions by users

### LINKED ISSUE #340: FAQ section missing from docs website
The Frequently asked questions, or FAQs, are a great way to improve the user's experience of the website. It allows to answer the questions that are most commonly asked. This section is missing from the website

### PR DESCRIPTION
**Pull request by salman-farooqi**
**What kind of change does this PR introduce?**
Added FAQ section to Talawa docs which provides answers to the Frequently Asked Questions by users. It will provide quick help to people visit the site, and also help rank the site in google SEO.

**Issue Number:**

Fixes #340 

**Did you add tests for your changes?**
I tested the result, it worked perfectly on my local server.

**Snapshots/Videos:**
faq
faq2

**If relevant, did you update the documentation?**
Yes, I updated the documentation. https://docs.talawa.io/docs/faq-section

**Summary**
See above.

**Does this PR introduce a breaking change?**
N/A

**Other information**
N/A

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by salman-farooqi** [comment]
Please check the result @palisadoes @Cioppolo14

File: docusaurus.config.js [hunk 1]
```diff
@@ -75,6 +75,12 @@ const config = {
                     label: "Talawa API",
                     position: "left",
                 },
+                {
+                  to: '/docs/faq-section',
+                  target: "_self",
+                  label: "FAQ",
```
Comment (palisadoes) [inline docusaurus.config.js]: Why was this edited?

File: docusaurus.config.js [hunk 2]
```diff
@@ -109,6 +115,11 @@ const config = {
                 to: "https://www.palisadoes.org/contact/",
                 className: "footer__icon footer__contact",
               },
+              {
+                label: ' FAQ',
+                to: '/docs/faq-section',
```
Comment (palisadoes) [inline docusaurus.config.js]: Why do we need this? There will be a FAQ in the header

File: src/css/custom.css [hunk 3]
```diff
@@ -248,6 +248,10 @@ h1.docTitle_node_modules-\@docusaurus-theme-classic-src-theme-DocItem- {
     mask: url(/img/team.svg) no-repeat 100% 100%;
     mask-size: cover;
 }
+.footer__faq:before {
+    mask: url(/img/faq.svg) no-repeat 100% 100%;
```
Comment (palisadoes) [inline src/css/custom.css]: Why was this added to the footer css?

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please answer the questions

**Comment by salman-farooqi** [comment]
Please check the result @palisadoes @Cioppolo14, is it good enough?
I need assistance in finding the answers for some of the questions.
I'm ready for further changes, if requested. Thanks!

File: docusaurus.config.js (hunk 1 above)
Comment (salman-farooqi) [reply docusaurus.config.js]: The changes are made to add a link in the header for FAQ section, that will redirect users to 'Introduction>FAQ' item, below the 'About Us' item.

File: docusaurus.config.js (hunk 2 above)
Comment (salman-farooqi) [reply docusaurus.config.js]: It seamed reasonable to me to add the link for FAQ both in the header and footer, as it's the most important section for the users to catch their attention.
But, if it's not required and suggested by mentors/maintainers, I'm ready to remove it. Thanks!

File: src/css/custom.css (hunk 3 above)
Comment (salman-farooqi) [reply src/css/custom.css]: It's the svg icon for the footer 'FAQ' link.
If FAQ link in the footer is suggested to remove by mentors, these CSS changes will be discarded too.

**Comment by palisadoes** [comment]
1. Please remove the FAQ from the footer. It is unnecessary as it is already in the header of your PR.
2. Duplicate links on the same page will reduce our SEO.

**Comment by salman-farooqi** [comment]
> 1. Please remove the FAQ from the footer. It is unnecessary as it is already in the header of your PR.
> 2. Duplicate links on the same page will reduce our SEO.

Done! Removed the FAQ from the footer, and thanks for the correction and guidance.

**Comment by Vic-Orlands** [comment]
> Please check the result @palisadoes @Cioppolo14, is it good enough?
> I need assistance in finding the answers for some of the questions.
> I'm ready for further changes, if requested. Thanks!

I will help out on the answers Salman. You will get them by tomorrow in the slack channel(under your thread)

**Comment by Cioppolo14** [comment]
@salman-farooqi Please address the conflicting files.

**Comment by salman-farooqi** [comment]
> @salman-farooqi Please address the conflicting files.

Sorry, I had to close the PR, becasue of the huge amount of changes made to the Website. I'll link a new PR.
